app/enterprise/1.5.x/kong-for-kubernetes/install.md

```diff
@@ -158,13 +152,223 @@ $ export PROXY_IP=$(kubectl get -o jsonpath="{.status.loadBalancer.ingress[0].ip
+## Option 2: Install Kong Enterprise on Kubernetes
 
-## Optional: Installing Kong Enterprise on Kubernetes
+The steps in this section show you how to install Kong Enterprise on Kubernetes using Helm.
+>Note: the following instructions assume that you're running Helm 3.
 
-> Note: The recommended installation is [Kong for Kubernetes Enterprise](#introduction).
+<img src="https://doc-assets.konghq.com/kubernetes/Kong-Enterprise-on-Kubernetes.png">
 
-To install Kong Enterprise on Kubernetes, instead of installing Kong for Kubernetes, see the following diagram. (Steps to be provided at a later time.)
+### Set up Helm
 
-<img src="https://doc-assets.konghq.com/kubernetes/Kong-Enterprise-on-Kubernetes.png">
+1. Add the Kong charts repository:
+    ```
+    $ helm repo add kong https://charts.konghq.com
+    ```
+2. Update Helm:
+    ```
+    $ helm repo update
+    ```
+
+### Configure Kong Enterprise Docker registry access
+Set up Docker credentials to allow Kubernetes nodes to pull down the Kong Enterprise Docker image, which is hosted as a private repository. You receive credentials for the Kong Enterprise Docker image when you sign up for Kong Enterprise.
+
+{% navtabs %}
+{% navtab kubectl %}
+1. Set up the credentials:
+    ```
+    $ kubectl create secret -n kong docker-registry kong-enterprise-edition-docker \
+        --docker-server=kong-docker-kong-enterprise-edition-docker.bintray.io \
+        --docker-username=<your-bintray-username> \
+        --docker-password=<your-bintray-api-key>
+    ```
+
+2. (Optional) Create a password for the super admin:
+    ```
+    kubectl create secret generic kong-enterprise-superuser-password  -n kong --from-literal=password=<your-password>
+    ```
+    Though not required, this is recommended for users that wish to use RBAC, as it cannot be done after initial setup.
+{% endnavtab %}
+{% navtab OpenShift oc %}
+1. Set up the credentials:
+    ```
+    $ oc create secret -n kong docker-registry kong-enterprise-edition-docker \
+        --docker-server=kong-docker-kong-enterprise-edition-docker.bintray.io \
+        --docker-username=<your-bintray-username> \
+        --docker-password=<your-bintray-api-key>
+    ```
+
+2. (Optional) Create a password for the super admin:
+    ```
+    oc create secret generic kong-enterprise-superuser-password -n kong --from-literal=password=<your-password>
+    ```
+    Though not required, this is recommended for users that wish to use RBAC, as it cannot be done after initial setup.
+{% endnavtab %}
+{% endnavtabs %}
+### Enable the sessions plugin for Manager and Dev Portal
+In the following steps, replace `<your-password>` with a secure password.
+
+{% navtabs %}
+{% navtab kubectl %}
+
+1. Create a sessions config file for Kong Manager:
+    ```
+    $ cat > admin_gui_session_conf
```

Comment:
This should be:
` $ cat admin_gui_session_conf`

Reply:
Wouldn't this just print the contents of the file? If it doesn't exist yet, it just errors out.

Reply:
I see what you meant. I was executing `cat > admin_gui_session_conf` while we want the user to execute the multi-line command. This _could_ confuse other users too.

if we are doing this in other places in the site, let's keep it as.
If not, then does it make sense to ask users to create two files themselves, and show only the content for them?

Reply:
echo allows this to use a oneliner if we prefer:
```
echo '{"cookie_name":"admin_session","cookie_samesite":"off","secret":"<your-password>","cookie_secure":false,"storage":"kong"}' > admin_gui_session_conf
```